charts/metal-control-plane/Chart.yaml

```diff
@@ -3,4 +3,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for deploying the metal control plane in K8s
 name: metal-control-plane
-version: 0.4.7
+version: 0.4.8
```

Comment:
As this is breaking: wouldn't be `0.5.0` better?

Reply:
Yes, right. Done.